CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,85 @@
+Code of Conduct
```

Comment:
I'll use this as a general comment. Given markdown is the norm for github docs and rst is the norm for sphinx docs, did you look at or consider integrating a markdown extension into sphinx? Such as, https://www.sphinx-doc.org/en/master/usage/markdown.html . I'm asking if this is an alternative to moving all the .md to .rst but note I'm not requiring you use this but wanted to have a conversation on alternatives. Note: the README.rst file was changed from .md awhile back to have been rendering inside of PyPI.

Reply:
To be frank, no I didn't consider it and I'm not opposed to doing it that way.  I tend to be a purist and as such generally avoid mixing of different markdowns, yaml and json or Python and bash, etc, this is for various reasons; some valid, some not.  While not native rST is "well" supported on both GitHub and GitLab and the purist in me would prefer to keep to one markdown language and to stick to the Sphinx way of doing things.  It also avoids yet one more dependency.  That said rST is not as well known as md and it might be easier for new contributors to work with.  So, I'll yield to your preference.  Let me know your decision and I'll resolve any issues.

Reply:
I chatted with @phobologic and he was a +1 on converting to rst. Let's keep that part of your commit and I'll await your commit later to revert the basestring changes. Thanks!